src/App.tsx

```diff
@@ -1,19 +1,113 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect, useMemo, useRef, useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
 
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Error } from './components/Error/Error';
+
+import { Todo } from './types/Todo';
+import { TypeError } from './types/TypeError';
+import { FilterType } from './types/FilterType';
+import { NewTodoField } from './components/NewTodoField/NewTodoField';
+import { getTodos } from './api/todos';
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const [hasError, setHasError] = useState(false);
+  const [errorType, setErrorType] = useState<TypeError>(TypeError.NONE);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.All);
+
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  async function LoadTodos() {
```

Comment:
`loadTodos`
use `camelCase` for function names. You can use Capitalize in React Component names and in types only.